packages/storybook/stories/INTERNAL/Survey/snippets/StarSurvey.snippet.tsx

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PrimaryButton } from '@fluentui/react';
+import { _StarSurvey as StarSurvey } from '@internal/react-components';
+import React, { useState } from 'react';
+
+export const StarSurveyExample: () => JSX.Element = () => {
+  const [showSurvey, setShowSurvey] = useState(false);
```

Comment:
Are these components going to be in the regular storybook at some point?

Reply:
They are going to be internal components only